addons/stock_account/tests/test_reconciliation.py

```diff
@@ -0,0 +1,130 @@
+                       "product_product_4")
+        # Configure the product to generate journal from picking
+        # Set a special cost to test the rounding
+        self.product_id.write({'standard_price': 84.4628,
+                               'valuation': 'real_time',
+                               'cost_method': 'standard'})
+        # Set required accounts in the category
+        self.product_id.categ_id.write({
+            'property_stock_account_input_categ': acc_in_id,
+            'property_stock_account_output_categ': acc_out_id,
+            })
+
+    def test_CostJournal(self):
+        '''
+        Test the total in the credit and debit amount from
+        account move line created from a picking with a product
+        with Real Time valuation
+        '''
+        cr, uid = self.cr, self.uid
+        # Creating picking out with a product with real time valuation
+        pick_id = self.picking.\
+            create(cr, uid,
+                   {
+                       'picking_type_id': self.pick_type,
+                       'partner_id': self.partner_id,
+                       'origin': 'Test Journal Items',
+                       'invoice_state': '2binvoiced',
+                       'move_lines': [(0, 0,
+                                       {
+                                           'product_id': self.product_id.id,
+                                           'name': self.product_id.name,
+                                           'price_unit': self.product_id.
+                                           standard_price,
+                                           'product_uom': self.product_id.
+                                           uom_id.id,
+                                           'product_uom_qty': 2,
+                                           'location_id': self.loc_id,
+                                           'location_dest_id': self.loc_dest_id
+                                       })]
+                   })
+        # Checking if the picking was created
+        self.assertTrue(type(pick_id) in (long, int),
+                        'The picking was not created correctly')
+        # Validating the picking to create the journal item
+        self.picking.action_confirm(cr, uid, [pick_id])
+        self.picking.force_assign(cr, uid, [pick_id])
+        self.picking.do_transfer(cr, uid, [pick_id])
+        pick_brw = self.picking.browse(cr, uid, pick_id)
+        # Verifying if the picking was validated
+        self.assertTrue(pick_brw.state == 'done',
+                        'The picking was not transfered')
+        # Creating the invoice from the picking
+        value = self.stock_invoice.\
+            default_get(cr, uid, [],
+                        {'active_model': 'stock.picking',
+                         'active_id': pick_id,
+                         'active_ids': [pick_id]})
+        invo_id = self.stock_invoice.create(cr, uid, value,
```

Comment:
Just being picky here, but I would use wiz_id instead of invo_id, as already done in elsewhere: https://github.com/odoo/odoo/blob/8.0/addons/stock_invoice_directly/test/stock_invoice_directly.yml#L44